service/src/test/java/datawave/microservice/query/executor/ShouldGenerateResultsTest.java

```diff
@@ -0,0 +1,138 @@
+package datawave.microservice.query.executor;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
This  should probably be jupiter Assertions for consistency

Reply:
you're right